adadamp/_dist.py

```diff
@@ -13,7 +13,7 @@
 from distributed import get_client
 from sklearn.base import BaseEstimator
 from sklearn.utils import check_random_state
-from skorch import NeuralNet
+# from skorch import NeuralNet
```

Comment:
Had issues while skortch was imported. Commented out this line and all was well. What was this import doing?